.github/workflows/phpunit-tests.yml

```diff
@@ -0,0 +1,144 @@
+name: PHPUnit Tests
+
+# Since Unit Tests are required to pass for each PR,
+# we cannot disable them for documentation-only changes.
+on:
+    pull_request:
+    push:
+        branches:
+            - trunk
+            - 'release/**'
+            - 'wp/**'
+
+# Cancels all previous workflow runs for pull requests that have not completed.
+concurrency:
+    # The concurrency group contains the workflow name and the branch name for pull requests
+    # or the commit hash for any other events.
+    group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.sha }}
+    cancel-in-progress: true
+
+jobs:
+    # Runs the PHPUnit tests for Gutenberg.
+    #
+    # Performs the following steps:
+    # - Sets environment variables.
+    # - Checks out the repository.
+    # - Sets up Node.js.
+    # - Sets up PHP.
+    # - Installs Composer dependencies.
+    # - Installs npm dependencies
+    # - Logs general debug information about the runner.
+    # - Logs Docker debug information (about the Docker installation within the runner).
+    # - Starts the wp-env Docker containers.
+    # - Logs the running Docker containers.
+    # - Logs debug information about what's installed within the wp-env Docker containers.
+    # - Install WordPress and Gutenberg within the Docker container.
+    # - Run the PHPUnit tests.
```

Comment:
Same remark about not duplicating information already available.

Reply:
Removed in https://github.com/WordPress/gutenberg/commit/2363c402862adec91e8429a53dd18756e228b990